### ISSUE TITLE
Question - How do I install Calico add-on in an EKS cluster?

### ISSUE THREAD

**Issue by vikas027** [tigera/operator#2026]
I am looking to install Calico add-on in an EKS cluster along with AWS VPC CNI as suggested in the official calico [documentation](https://projectcalico.docs.tigera.io/getting-started/kubernetes/managed-public-cloud/eks#install-eks-with-amazon-vpc-networking) which points to the AWS official [documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html), which further states that the recommended way of installing Calico on EKS is by using this [operator](https://github.com/tigera/operator).

I am not sure how to do the same with the operator. 

What I have tried :)
- look in the certification courses but that too has the old way of installing it

Alternatives
- Helm Installation `helm install calico projectcalico/tigera-operator --version v3.23.1`

**Comment by tmjd** [comment]
Yeah it isn't clear how to do this.
I think you should be able to follow the [Calico quickstart](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart) directions and just ensure you properly configure the `custom-resources.yaml` for the AWS CNI plugin (after you install the AWS CNI plugin).

**Comment by caseydavenport** [comment]
The Amazon EKS documentation has steps for installing Calico listed here: https://docs.aws.amazon.com/eks/latest/userguide/calico.html#calico-install

Looks like it could use an update to the latest version, though.

**Comment by vikas027** [comment]
> The Amazon EKS documentation has steps for installing Calico listed here: https://docs.aws.amazon.com/eks/latest/userguide/calico.html#calico-install
> 
> Looks like it could use an update to the latest version, though.

Yes, this is the link I have in my query as well. It does not state how to install using the operator.

**Comment by caseydavenport** [comment]
@vikas027 I see that the doc is not very clear, but the instructions listed under the "Helm" tab on that page are the correct instructions to install the tigera-operator (using helm), very similar to the instructions we have here: https://projectcalico.docs.tigera.io/getting-started/kubernetes/helm

We should clean up the language on the Amazon doc so that it's less confusing, but the content there is correct.
